src/pages/NightCream/NightCream.js

```diff
@@ -0,0 +1,101 @@
+import './NightCream.css';
+import placeholder from '../../assets/placeholder.jpg';
+function NightCream() {
+  return (
+    <div className="night-cream">
+      <div className="back-button">
+        <a href="/">
+          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="42" viewBox="0 0 24 42" fill="none">
+            <path d="M2.14648 20.5137L22.0001 2.0001" stroke="black" stroke-width="3" stroke-linecap="round" />
+            <path d="M2 20.6558L21.8286 39.9997" stroke="black" stroke-width="3" stroke-linecap="round" />
+          </svg>
+        </a>
+      </div>
+      <div className="product">
+        <div className="product-images">
+          <img src={placeholder} alt="Night Cream 1" />
+          <img src={placeholder} alt="Night Cream 2" />
+          <img src={placeholder} alt="Night Cream 3" />
+        </div>
+        <div className="product-details">
+          <div className="product-info">
+            <h2>Night Cream</h2>
+            <p>"Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore</p>
+            <div className='product-ingredients'>
+              Ingredients:&nbsp;
+              <a href="/">Oil</a>,&nbsp;
```

Comment:
same thing as with the stars - if the list of ingredients is stored as a list, you can iterate over that list and render an <a> for each of them